app/models/container_project.rb

```diff
@@ -78,4 +78,8 @@ def disconnect_inv
     self.deleted_on = Time.now.utc
     save
   end
+
+  def archived?
+    ems_id.nil?
+  end
```

Comment:
@simon3z Shamelessly copied from `VmOrTemplate` [here](https://github.com/zeari/manageiq/blob/d83a8b0b16a9d9816f4706e83038a609f4daa57c/app/models/vm_or_template.rb#L1373-1373)

I think this way of checking for disconnection is ok for the containers area too.